lib/src/main/java/com/alphawallet/token/entity/ContractAddress.java

```diff
@@ -10,17 +10,27 @@
 {
     public final int chainId;
```

Comment:
isEnabled needs to be removed from this class. It should only be in the TokenMeta

Reply:
removed